contracts/hardhat.config.js

```diff
@@ -12,6 +12,7 @@ require("hardhat-deploy-ethers");
 require("hardhat-gas-reporter");
 require("solidity-coverage");
 require("@openzeppelin/hardhat-upgrades");
+require("hardhat-gas-trackooor");
```

Comment:
It looks like this plugin hasn't been updated in about 18 months. The GitHub repo has no activity either. Would be better if we can use something that's being actively maintained